src/test/java/core/basesyntax/service/ReaderServiceTest.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.util.List;
+import org.junit.Test;
+
+public class ReaderServiceTest {
+    private static final ReaderService READER = new ReaderServiceImpl();
```

Comment:
Let's initialize classes being tested in a separate method annotated with '@BeforeClass'. Also it shouldn't be uppercased, as it's a service, not a constant. Please fix it in all places